zeebe/bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/validation/ZeebeTaskValidatorFormTest.java

```diff
@@ -625,7 +639,21 @@ public static Object[][] parameters() {
             .endEvent()
             .done(),
         EMPTY_LIST
-      }
+      },
+      {
+        Bpmn.createExecutableProcess("process")
+            .startEvent()
+            .userTask("task")
+            .zeebeUserTask()
+            .zeebeExternalFormReference("reference")
+            .zeebeFormBindingType(ZeebeBindingType.deployment)
+            .endEvent()
+            .done(),
+        singletonList(
+            expect(
+                ZeebeFormDefinition.class,
+                "Attribute 'externalReference' must not be used in combination with attribute 'bindingType'"))
```

Comment:
🔧 We could consider adding a test that shows what happens when a string is used that is not an enum value of `ZeebeBindingType` as `bindingType`

Reply:
Good suggestion 👍🏻. I added such tests with https://github.com/camunda/camunda/pull/19766/commits/9efc3ee74a44974b1ce19e615a2dc08b881c087c.